### ISSUE TITLE
Aminothiazoles: Promiscuity and Reactive metabolites

### ISSUE THREAD

**Issue by bendndi** [OpenSourceMycetoma/Series-2-Aminothiazoles#1]
A couple of thoughts on issues associated with 2-aminothiazoles 

1) 2-aminothiazoles are known to undergo oxidative metabolism (epoxidation) to reactive metabolites; see [this pape](https://pubs.acs.org/doi/abs/10.1021/tx900414g)r for an example. There are strategies to avoid this (substitution of both the 4 and 5 position; change to diaminothiazoles...), however this is certainly something to keep in mind as we move forward

2) 2-aminothiazoles come up [everywhere](https://pubs.acs.org/doi/10.1021/jm501402x), and this series is a frequent hitter in most DNDi screens. It's not necessarily an assay interference issue though (it's a frequent positive, not necessarily a frequent false positive), and there are indeed launched drugs with similar motifs, but again something to keep in mind.

**Comment by drc007** [comment]
@bendndi I agree, in which case if we can identify the mode of action alternative chemotypes might be attractive.

**Comment by MFernflower** [comment]
It would be best to screen a chemical that is not of the aminothiazole family but still blocks this mysterious potassium channel
